src/clients/Client.h

```diff
@@ -155,6 +155,10 @@ class Client:
     size_t calcBufferSpaceToReserve(const size_t space, const size_t wantSpace) const;
 
     void adjustBodyBytesRead(const int64_t delta);
+    /// A callback used by delayRead(); implement in sublclasses calling delayRead().
+    virtual void noteDelayedRead() = 0;
+    /// Defer a noteDelayedRead() call, which must be implemented in subclasses.
```

Comment:
The goal here is probably to defer a read; deferring the call is just a (part of the) mechanism doing that. Let's also add a new line, for readability sake.

```suggestion

    /// Defer reading until it is likely to become possible.
    /// Eventually a noteDelayedRead() callback will be called.
```
Please double check that my "likely to become possible" condition is more-or-less accurate. We do not have to be super precise here -- there are probably a lot of variables that control the need for this delay and the timing of the callback -- but we need to give a general idea about what is going on.